benchmark.py

```diff
@@ -150,48 +165,55 @@ def main():
   parser.add_argument("output_dir", help="Output dir")
   parser.add_argument('--no-checks', dest='no_checks', action='store_true',
                     help="Invalid environment doesn't throw")
-  parser.add_argument('--profile', dest='run_profile', action='store_true',
-                    help="Run a profiling of the application for cpu use.")
   parser.add_argument('--measure', dest='run_measure', action='store_true',
                     help="Run measurments of the cpu use of the application.")
+
+  # Profile related arguments
+  parser.add_argument('--profile_mode', dest='profile_mode', action='store',
+          help="Run a profiling of the application in one of two modes: wakeups, cpu_time.")
   parser.add_argument('--dry_run', dest='dry_run', action='store_true',
                     help="Do not actually profile run commands but print them out.")
   parser.add_argument('--chromium_executable', dest='chromium_executable', action='store',
                     help="Absolute path to a locally built Chromium binary.")
+
   args = parser.parse_args()
 
   if args.run_measure and args.dry_run:
       print("Dry running measure is not implemented !")
       exit(-1)
 
-  if args.run_profile and args.run_measure:
+  if args.profile_mode and args.run_measure:
       print("Cannot measure and profile at the same time, choose one.")
       exit(-1)
+  
+  if args.profile_mode and args.profile_mode not in ["wakeups", "cpu_time"]:
```

Comment:
Argparse has a "choices" argument that can do this, and is included in help option.

Reply:
Cool! Less code :)